curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6553efd6ada3f42aa2d75448.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 6553efd6ada3f42aa2d75448
+title: Step 48
+challengeType: 20
+dashedName: step-48
+---
+
+# --description--
+
+Now you should see the output again. Although this approach works, it doesn't significantly enhance the code's reusability. Repeatedly calling your function yields the same outcome. However, functions can be declared with *parameters* to introduce versatility and customization:
```

Comment:
I would suggest updating "yields" to a different word because [yield](https://docs.python.org/3/reference/expressions.html#yield-expressions) is a type of expression in Python used to define generator functions.

 ```suggestion
Now you should see the output again. Although this approach works, it doesn't significantly enhance the code's reusability. Repeatedly calling your function will result in the same outcome. However, functions can be declared with *parameters* to introduce versatility and customization:
```